stackslib/src/net/stackerdb/config.rs

```diff
@@ -221,114 +219,64 @@ impl StackerDBConfig {
             burn_dbconn,
             tip,
             contract_id,
-            "({STACKERDB_SLOTS_FUNCTION})",
+            "(stackerdb-get-signer-slots)",
```

Comment:
For some reason, when I tried to run this code, the VM evaluated the literal string `({STACKERDB_SLOTS_FUNCTION})` and rightfully complained about the lack of a `:` in the tuple.  Not sure why, because it was imported, so I just replaced it with the function name.

I'll try again.